structuredevent-service-cdp/src/main/java/com/sequenceiq/cloudbreak/structuredevent/service/telemetry/converter/EnvironmentDetailsToCDPNetworkDetailsConverter.java

```diff
@@ -88,4 +92,18 @@
         }
         return cdpProxyDetailsBuilder.build();
     }
+
+    private UsageProto.CDPOwnDnsZones convertOwnDnsZones(NetworkDto networkDto) {
```

Comment:
please add unit test for this in EnvironmentDetailsToCDPNetworkDetailsConverterTest